synapse/tests/test_tools_genpkg.py

```diff
@@ -78,7 +82,7 @@ async def test_tools_genpkg(self):
             s_common.yamlsave(pdef, yamlpath)
 
             self.eq(pdef['name'], 'testpkg')
-            self.eq(pdef['version'], (0, 0, 1))
```

Comment:
why does this need to change?

Reply:
I changed this because we norm versions in the cortex to strings, and expect versions to be strings for validation, but genpkg was converting them to tuples. Seemed better to just have them always be strings so we can validate and not be flipping it back and forth.